### DISCUSSION TITLE
GENE expression

### DISCUSSION THREAD

**Discussion by SaraSALHI2** [Open-Systems-Pharmacology/Forum#1369] [Questions & Problems]
Dear All,

I downloaded GENEDB_human.expressionDb and entered it in the utilities_option_application however I can't see the database 
![screenshot](https://user-images.githubusercontent.com/116799648/198247202-7c8dff81-497b-44fa-888a-88a61273430d.png)

**Comment by Yuri05** [comment]
Please see the answer to your question in this recent discusion: https://github.com/Open-Systems-Pharmacology/Forum/discussions/1359

**Reply by SaraSALHI2** [reply]
Thank you very much, I just noticed the answer
